### PR TITLE
unistd imports fix

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
There are several unistd imports left in the sources that needed to be fixed in order to be built on windows.

### CODE REVIEW

**Comment by JPTIZ** [comment]
I thought I handled them in #149.

**Comment by joaoantoniocardoso** [comment]
> I thought I handled them in #149.

Possibly yes, do you have a commit or PR that isolates your unistd fix so I can use in other branches instead of using this one?

**Comment by Coquinho** [comment]
@JPTIZ partially, not all changes done here, happen at #149, `eet` for example.

File: src/lib/ecore/ecore_anim.c [hunk 1]
```diff
@@ -4,21 +4,22 @@
...
 #include <math.h>
-#include <unistd.h>
 #include <string.h>
-#include <sys/types.h>
-#include <sys/stat.h>
+
+#ifndef _WIN32
+# include <unistd.h>
+# include <sys/types.h>
+# include <sys/stat.h>
```
Comment (Coquinho) [inline src/lib/ecore/ecore_anim.c:12]: `ucrt` has `sys/types.h` and `sys/stat.h`

File: src/lib/ecore/ecore_time.c [hunk 2]
```diff
@@ -5,7 +5,11 @@
 #define EFL_LOOP_PROTECTED
 
 #include <stdlib.h>
-#include <sys/time.h>
+#ifndef _WIN32
+# include <sys/time.h>
+#else
+# include <evil_private.h>
```
Comment (Coquinho) [inline src/lib/ecore/ecore_time.c:11]: why evil is needed here?

File: src/lib/ecore/efl_thread.c [hunk 3]
```diff
@@ -7,17 +7,22 @@
 #define EFL_IO_CLOSER_PROTECTED 1
 
 #ifdef _WIN32
-# include <evil_private.h> /* pipe fcntl */
```
Comment (Coquinho) [inline src/lib/ecore/efl_thread.c:10]: those guards are still here.

File: src/lib/ecore/ecore_main.c [hunk 4]
```diff
@@ -13,12 +13,17 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
-#include <unistd.h>
 #include <math.h>
-#include <sys/types.h>
 #include <errno.h>
+#include <sys/types.h>
```
Comment (Coquinho) [inline src/lib/ecore/ecore_main.c:18]: I think we should let `sys/types` where it was, I don't know if phab will like this latter.

File: src/lib/ecore/ecore_main_timechanges.c [hunk 5]
```diff
@@ -5,12 +5,17 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
-#include <unistd.h>
 #include <math.h>
-#include <sys/types.h>
```
Comment (Coquinho) [inline src/lib/ecore/ecore_main_timechanges.c:10]: Here `sys/types` is still removed

**Pull request merged**